src/Twig/WishlistExtension.php

```diff
@@ -56,16 +60,20 @@ public function getWishlists()
 
     public function findAllByShopUser()
```

Comment:
This should accept `UserInterface $user = null` as it's arguments, why it has so much logic?